packages/runtime/src/runtime.ts

```diff
@@ -422,6 +426,30 @@ export class Runtime {
 		this.store.accounts.set(feeSink.address, feeSink);
 	}
 
+	/**
+	 * Add accounts from config file to the Store
+	 * @param network: the network of accounts to add
+	 * @param balance: balance for accounts
+	 */
+	loadAccountsFromConfig(network = "default", balance?: number): void {
+		const find = findupSync([JS_CONFIG_FILENAME, TS_CONFIG_FILENAME]);
+		if (!find) {
+			throw new RuntimeError(RUNTIME_ERRORS.GENERAL.CONFIG_FILE_NOT_FOUND);
+		}
+		// eslint-disable-next-line @typescript-eslint/no-var-requires
+		const config = require(find);
+
+		console.log(config);
+		console.log(config.networks[network]);
+		const accounts = config.networks[network].accounts;
```

Comment:
what happens if the network user provided is not defined?

Reply:
Just add new error check if network exist or not